MapboxNavigation/ImageDownloader.swift

```diff
@@ -56,10 +61,15 @@ class ImageDownloader: NSObject, ReentrantImageDownloader, URLSessionDataDelegat
     }
 
     func activeOperationWithURL(_ url: URL) -> ImageDownload? {
```

Comment:
While we’re here, would you mind changing the method name to `activeOperations(url:)`? The “with” followed by `_` is weirding me out.

Reply:
👍